packages/ocs/storage-class/sc-form.scss

```
@@ -56,3 +56,8 @@
     padding: 0 0 var(--pf-v5-global--spacer--md) var(--pf-v5-global--spacer--md);
   }
 }
+
+.ocs-storage-class__pool--create {
```

`--something` is for CSS property...
```suggestion
.ocs-storage-class__pool-create {
```

---

@SanjalKatiyar It's correct according to [BEM naming](https://getbem.com/naming/): it's a modifier.